src/blocks/author.ts

```diff
@@ -0,0 +1,20 @@
+import { Author } from "./types";
+
+
+const DEFAULT_AUTHOR: Author = {
+  id: '',
+  name: null,
+  userId: null,
+  orcid: null,
+  corresponding: null,
+  email: null,
+  roles: [],
+  affiliations: []
+}
+
+export function createAuthor(initialState: {id: string} & Partial<Author>): Author {
+  return {
+    ...DEFAULT_AUTHOR,
```

Comment:
Can we clone the arrays of role/affiliation or leave them as null?

Reply:
Cloning arrays in unnecessary imo. We shouldn't rely on mutating the arrays anyway. Have you encountered cases DTO functions requires deep cloning?